### ISSUE TITLE
Odroid U3 Ubuntu/Debian - Missing VLAN 8021q module

### ISSUE THREAD

**Issue by AKA-47** [hexdump0815/imagebuilder#48]
Hi.
Im using the image on odroid u3 using kernel 5.10.45-stb-exy+

Installing the vlan package via apt doesnt work to get vlans running because the 8021q kernel module is missing.

Do you have any advise to get this running?

**Comment by hexdump0815** [comment]
i think the best would be to build your own kernel directly following my notes from https://github.com/hexdump0815/linux-mainline-and-mali-generic-stable-kernel/blob/master/readme.exy - it is not that complicated as long as there is some space on the sd card

best wishes and good luck - hexdump

**Comment by Silverstar** [comment]
I was able to use the official kernel provided by the backports repository on my u1. 
I needed that because I was missing kernel modules as well and didn't want to compile a kernel on my own so I tried.
You will have to edit the extlinux.conf in the boot partition to use another kernel.
See in my issue #39
But I don't know what else might not work anymore with that kernel, for my tasks everything works.
